web-docs/docs/02-CHANGELOG.mdx

```diff
@@ -5,6 +5,19 @@ sidebar_label: Change Log
 toc_max_heading_level: 2
 ---
 
+## Version 1.15.0
+### Features
+- Added data aggregation feature to historical queue.
+- Added date conversion to ISO-8601 formatted string utility method in SCTimeUtils.java.
+- Added combined logging methods for log messages and exceptions in Logger.java.
+- Added support for time units up to years in SCTimeSpan.java.
+- Added methods for converting one (1) unit of time to another in a more friendly manner in
+  SCTimeUnit.java.
+- Updated base project files to use the latest from the upstream starter project.
+  - Documentation is now generated using Docusaurus.
```

Comment:
Yes, I can remove the indent if you think it makes more sense. I just indented this since the documentation change is mainly a side effect of updating base project files.

Reply:
Since you added documentation and modified it to fit this repo it seems like its its own feature.

Maybe keep that line and add
```suggestion
  - Documentation is now generated using Docusaurus.
- Adapted documentation to Docusaurus. 
```

Reply:
Changed